### PR TITLE
Multiple unnecessary dom updates

### PR DESCRIPTION
**Pull request by sajov**
https://github.com/remy/bind.js/issues/13

### CODE REVIEW

**Comment by remy** [comment]
Can you fix the white space to match the original code (2 spaces, not tabs) so I can diff and (hopefully) merge? Thank you! <3

**Comment by sajov** [comment]
ok, white space are fixed

**Comment by remy** [comment]
I'm struggling a little to get my head around exactly what you're saying (I'm sorry).

I looked at your test jsbin, and I can see the multiple firing updates, so it makes sense to fix that and only allow the update to fire once.

I think it would also be worth adding a test that shows this bug being fixed. 

That same test could perhaps be used to show the "other issue" that you're [referring to](https://github.com/remy/bind.js/pull/14#issuecomment-153522318).

File: lib/bind.js [hunk 1]
```diff
@@ -398,7 +398,9 @@ var Bind = (function Bind(global) {
             findCallback(value);
           } else {
             // defer the callback until we're fully booted
-            settings.deferred.push(findCallback.bind(target, value, old));
+            if (typeof settings.mapping[path.join('.')] != 'undefined') {
```
Comment (remy) [inline lib/bind.js]: Looks good, can you just update `!=` to `!==` as per the `.jscsrc` file?

**Comment by remy** [comment]
Hoping this PR is still active, can you add the test I asked for in the previous comment?

**Comment by remy** [comment]
I've merged this change in with v1.1.0.

Thanks again - though I'm not quite sure how it worked, it did :)

**Pull request closed without merging**